proc/grouper.go

```diff
@@ -3,8 +3,8 @@ package proc
 import (
 	"time"
 
+	common "github.com/finbourne/process-exporter"
```

Comment:
are the import path changes required?

Reply:
Yes, otherwise older versions (i.e. upstream) of these packages will be used which do not have the new functionality in this PR.